docs/sources/next/javascript-api/k6-experimental/browser/element-handle/dblclick.md

```diff
@@ -0,0 +1,58 @@
+| ------------------- | -------- | ------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| options             | object   | `null`  |                                                                                                                                                                                                                                                                                                                                               |
+| options.button      | string   | `left`  | The mouse button (`left`, `middle` or `right`) to use during the action.                                                                                                                                                                                                                                                                      |
+| options.delay       | number   | `0`     | Milliseconds to wait between `mousedown` and `mouseup`.                                                                                                                                                                                                                                                                                       |
+| options.force       | boolean  | `false` | Setting this to `true` will bypass the actionability checks (`visible`, `stable`, `enabled`).                                                                                                                                                                                                                                                 |
+| options.modifiers   | string[] | `null`  | `Alt`, `Control`, `Meta` or `Shift` modifiers keys pressed during the action. If not specified, currently pressed modifiers are used.                                                                                                                                                                                                         |
+| options.noWaitAfter | boolean  | `false` | If set to `true` and a navigation occurs from performing this action, it will not wait for it to complete.                                                                                                                                                                                                                                    |
```

Comment:
```suggestion
| options.noWaitAfter | boolean  | `false` | If set to `true` and navigation occurs while performing this action, it doesn't wait for it to complete.        
```

Reply:
Resolved in 0abcf69969a3e825f97446543ddf9575c3bf0ff0